toc.yml

```diff
@@ -196,6 +196,9 @@
       - name: Network Programming in the .NET Framework
         tocHref: /dotnet/framework/network-programming/
         topicHref: /dotnet/framework/network-programming/index
+      - name: .NET Framework Deployment
```

Comment:
@mairaw Seems a little strange ... "Development Guide" (the topic) content seems like it should be an index page inside a folder that holds the rest of the folders it covers. In the TOC, it's the head of these dev topic groups; but in the file structure, it's just a topic in the ".NET Framework" group. Therefore, the .NET Framework group seems to suffer from a kind of *split-personality disorder!* :grinning: It's like there are *two* index pages in this group: The actual index and this Development Guide topic with all of the content of both "index" pages sitting side-by-side.

Anywho ..... so to fix, I just need to move the YML bits covered by Development Guide (that are currently under .NET Framework) to a new node under Development Guide that's under .NET Framework. :smile: Don't worry ... I'm sure it will make a lot more sense after I have a hard root beer. :beer: :grinning: